src/widgets/helper/ResizingTextEdit.cpp

```diff
@@ -158,6 +169,7 @@ void ResizingTextEdit::keyPressEvent(QKeyEvent *event)
             this->completionInProgress_ = true;
             this->completer_->setCompletionPrefix(currentCompletionPrefix);
             this->completer_->complete();
+            this->updatingText_ = false;
```

Comment:
This could use a signal blocker:

```suggestion
            {
                // block cursor position events
                QSignalBlocker b(this);
                this->completer_->complete();
            }
```